tests/integration/targets/win_acl/tasks/main.yml

```diff
@@ -16,6 +16,13 @@
   - absent
   - present
 
+- name: add PKI module to Powershell to create testing certificate
+  win_shell: Import-Module PKI
+
```

Comment:
d'oh - good point. I will say I think I put it in initially because the pipeline logs were saying the command `New-SelfSignedCertificate` wasn't found, but maybe I just had a typo and didn't realize it